gradle/libs.versions.toml

```diff
@@ -65,7 +65,7 @@ jmh-annotationprocessor = { group = "org.openjdk.jmh", name = "jmh-generator-ann
 jcstress-core = { group = "org.openjdk.jcstress", name = "jcstress-core", version.ref = "jcstress" }
 
 # Code Generation
-javaPoet = { group = "com.squareup", name = "javapoet", version.ref = "javaPoet" }
+javaPoet = { group = "com.palantir.javapoet", name = "javapoet", version.ref = "javaPoet" }
```

Comment:
I would prefer to stick with javapoet for the moment personally, we only use its output (which is very simple in our case) so it's not a security concern. I am a little worried about palantir's pre-1.0 downgrade as well as their decision to change the package name.

Reply:
To clarify (since both libraries are called javapoet), would you prefer I switch the library back to the original one from squareup?
If so, I'll have to change the generation slightly since the old library cannot generate records.

Reply:
square

Reply:
Reverted, though I'll open an issue regarding the archived library.